### ISSUE TITLE
Should be able to get new devices dialog again after dismissing it

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#3857]
### Description

When the new devices dialog shows up you have the option of "Send Anyway" or "Dismiss" (ok button). After dismissing you see something like this:
![image](https://cloud.githubusercontent.com/assets/1190097/25816577/bd31e2b2-33e1-11e7-881d-1bbeab4ed248.png)

In the status bar should be an option to show the dialog again, or at least have some way of knowing which devices are unknown.

### Steps to reproduce

- Have new devices in an encrypted room
- Send message
- Dismiss dialog

Log: not sent   

### Version information

- **Platform**: web (in-browser)
- **Browser**: Chrome 58
- **OS**: Windows 10
- **URL**: riot.im/develop

*Edited: It's a new devices dialog*

**Comment by richvdh** [comment]
Part of the problem here is that the act of showing you the dialog means that the devices are no longer new - so if you could see it again, it would be empty (as evidenced by the fact sending another message will not show the dialog).

Maybe we shouldn't mark the devices as 'known' when you 'dismiss' the dialog?

**Comment by Matrixcoffee** [comment]
The confusion lies in the combination of not sending the message, but still acknowledging the keys.

When users are presented with a binary choice in this way, they expect either "Do" or "Don't".

Having one option apparently be "Don't" but still have subtle side effects is indeed confusing.

As is having "Do" not send the message. When Android had this behaviour, we had many questions asking why the message went unsent.

So the user should either be presented with all three options, with clearly labeled buttons:

1. Cancel
2. Acknowledge, Hold Message
3. Acknowledge and Send

or (2) should be taken out as @richvdh said.

**Comment by ara4n** [comment]
i agree the UDD sucks. it feels like we should just show a dashboard for all the devices in a room when something changes (highlighting the change) so the user can choose how to react. we could also bring it up at any point from room settings or perhaps by hitting the padlock next to the composer.

The key thing would be to not just show deltas, but provide UI for generally managing how you treat the e2e devices in the room.

**Comment by ara4n** [comment]
merging this into #4522
